controllers/manila_controller.go

```diff
@@ -874,6 +896,21 @@ func (r *ManilaReconciler) generateServiceConfig(
 		"MemcachedServersWithInet": strings.Join(memcached.Status.ServerListWithInet, ","),
 	}
 
+	// create httpd  vhost template parameters
+	httpdVhostConfig := map[string]interface{}{}
+	for _, endpt := range []service.Endpoint{service.EndpointInternal, service.EndpointPublic} {
+		endptConfig := map[string]interface{}{}
+		endptConfig["ServerName"] = fmt.Sprintf("%s.%s.svc", manila.ServiceName, endpt.String())
```

Comment:
this should be `fmt.Sprintf("%s-%s.%s.svc", manila.ServiceName, endpt.String(), instance.Namespace)` , no?

Reply:
You're right, namespace was missing in th eoutput.